.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: Testing
+
+on: [push, pull_request]
+env:
+  EMG_CONFIG: ${{ github.workspace }}/ci/configuration.yaml
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: 🐍 - Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+    - name: 💾 Start MongoDB
+      uses: supercharge/mongodb-github-action@1.3.0
+      with:
+        mongodb-version: 4.0.6
+    - name: ⚙️ - Checking environment
+      run: |
+        python -V
+        sudo systemctl start mysql
+        mysql --version
+        mysql -uroot -proot -e 'CREATE DATABASE emg;'
+        mysql -uroot -proot -e "SET GLOBAL sql_mode = 'STRICT_TRANS_TABLES'"
+
+    - name: 🔧 - Install Dependencies
+      run: |
+        pip install -U git+git://github.com/EBI-Metagenomics/emg-backlog-schema.git
+        pip install -U git+git://github.com/EBI-Metagenomics/ena-api-handler.git
+        pip install -U -r requirements-test.txt
```

Comment:
Is it still required to install requirements-dev.txt? I remember needing to do this also.